### PR TITLE
Refactoring shared header code between games.

### PR DESCRIPTION
**Pull request by moniika**
Refactored shared header code between games. 
Generated html is unchanged with the exceptions:
- div with right nav elements has new id "header_cta"
- pond now has language selector
- pond header html structure has been modified (instead of only header, is now a table like other games).

### CODE REVIEW

File: appengine/common/common.css [hunk 1]
```diff
@@ -16,6 +16,9 @@ a:hover {
 .farSide {
   text-align: right;
 }
+#header_cta > * {
```
Comment (NeilFraser) [inline appengine/common/common.css]: Remove spaces around the > (to be consistent with similar rules in this file).

File: appengine/gallery/template.soy [hunk 2]
```diff
@@ -24,23 +24,14 @@
...
-            {/param}
-          {/call}
-        </h1>
-      </td>
-      <td class="farSide">
-        <select id="languageMenu"></select>
-      </td>
-    </tr>
-  </table>
+  {call .messages /}
```
Comment (NeilFraser) [inline appengine/gallery/template.soy]: Is a call to .messages needed here?

More importantly, is .messages needed at all?  Looks like dead code.

File: appengine/genetics/template.soy [hunk 3]
```diff
@@ -141,36 +141,14 @@
...
-            <img src="common/1x1.gif" class="link icon21">
-          </button>
-          &nbsp;
-          <button id="helpButton">
-            {{msg meaning="Games.help" desc="IBID"}}Help{{/msg}}
-          </button>
-        </td>
-      </tr>
-    </table>
+  {call .messages /}
```
Comment (NeilFraser) [inline appengine/genetics/template.soy]: Duplicate call to .messages

File: appengine/template.soy [hunk 4]
```diff
@@ -74,6 +74,56 @@
...
+            {call .levelLinks}
+              {param level: $ij.level /}
+              {param maxLevel: $ij.maxLevel /}
+              {param lang: $ij.lang /}
+              {param suffix}{if $levelLinkSuffix}{$levelLinkSuffix}{/if}{/param}
+            {/call}
+          {/if}
+        </h1>
+      </td>
+      <td id="header_cta" class="farSide">
```
Comment (NeilFraser) [inline appengine/template.soy]: Out of curiosity, what does cta stand for?  California Teachers Association?  Counter Terrorism Agency?

File: appengine/common/common.css (hunk 1 above)
Comment (moniika) [reply appengine/common/common.css]: Done.

File: appengine/gallery/template.soy (hunk 2 above)
Comment (moniika) [reply appengine/gallery/template.soy]: Looks like it. I'll remove the Movie.submitDisabled here, its defined in Movie already.

File: appengine/genetics/template.soy (hunk 3 above)
Comment (moniika) [reply appengine/genetics/template.soy]: Done.

File: appengine/template.soy (hunk 4 above)
Comment (moniika) [reply appengine/template.soy]: Call to action

**Pull request merged**